demos/2019_RHTE/keynote/create_rhte_env.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+if [[ -z "${AGNOSTICD_HOME}" ]]; then
+    echo "Please ensure that 'AGNOSTICD_HOME' is set before running."
+    exit
+fi
+
+OUR_DIR=`pwd`
+pushd .
+cd ${AGNOSTICD_HOME}
+
+echo "Creating OCP3 env..."
+ansible-playbook ${AGNOSTICD_HOME}/ansible/main.yml -e @${OUR_DIR}/3.x/my_vars.yml -e @${OUR_DIR}/3.x/ocp3_vars.yml -e @${OUR_DIR}/secret.yml
```

Comment:
Assuming these invocations are serial currently?
We'd want to invoke the creation of 3.x and 4.x envs in parallel.

Reply:
yes that makes sense

Reply:
@jwmatthews good to merge with https://github.com/fusor/mig-agnosticd/pull/55/commits/7de8486e24f298695cd7c579c0e758919eab9e1a